BUILD.md

```diff
@@ -23,5 +23,15 @@ To find the jar once it's built, just look at the last few lines of output from
 If you're going to make changes to the Scala code, you may want to use IntelliJ and it's Scala
 plugin to help with the Scala syntax...!
 
+If you use [Eclipse IDE](http://www.eclipse.org/), you can set-up your development environment by following these instructions:
+
+* Install `sbt` and build as-above
+* Install [Scala IDE for Eclipse](http://scala-ide.org/) into your Eclipse installation if not already installed
```

Comment:
Rather than include the plugin in `project/plugins.sbt`, point the users to https://github.com/typesafehub/sbteclipse#for-sbt-013-and-up and the global file install with `~/.sbt/0.13/plugins/plugins.sbt`